functional-tests/index.html

```diff
@@ -0,0 +1,27 @@
+<head>
```

Comment:
We'll probably want to make this page configurable so that our tests can determine things like
1. do we want annotations
2. do we want a header bar
3. is download enabled
etc.